rsp_jupyter_extensions/handlers/tutorials.py

```diff
@@ -382,5 +290,128 @@ def post(self) -> None:
                 else:
                     self.set_status(guide.status_code)
             return
-        self.log.info(f"Replying with dest = '{guide.dest}'")
+        self.log.debug(f"Replying with dest = '{guide.dest}'")
         self.write(json.dumps({"dest": guide.dest}))
+
+    def _get_github_tutorials(
+        self, dirname: str, *, from_cache: bool = False
+    ) -> Hierarchy:
+        homedir = _get_homedir()
+        tutorial_dir = homedir / "notebooks" / "tutorials"
+        if from_cache:
+            self.log.debug(f"get_gh: Checking cached repo {dirname}")
+            # Does it appear to be a git repo?
+            repo_git = Path(dirname) / ".git"
+            if not repo_git.is_dir():
+                self.log.debug("get_gh: Not a repo: force new clone")
+                from_cache = False  # force new clone
```

Comment:
This works fine as it is, but I'm wondering if it's better to use a separate local variable here like 
`use_cache = from_cache ` to make future refactoring a bit safer (We may miss the re-assignment of the function argument here)